src/app/services/task.service.ts

```diff
@@ -1,15 +1,16 @@
 import { Injectable } from '@angular/core';
```

Comment:
Just an architectural note, if I remember correctly it should be called `tasks-state-facade.service.ts`, just because it's simplify manipulating store. Also you can implement there simple version of selectors. For example, `tasks$ = this.store.select(tasksSelector)` and also if you choose this pattern you should put here a filtered tasks.